app/models/doc_capture.rb

```diff
@@ -1,10 +0,0 @@
-class DocCapture < ApplicationRecord
```

Comment:
Once this hits prod, we'll need to come back and drop the db table for this model.

Reply:
I opened #4519 to drop the table.